### PR TITLE
kafka: Export topics to external cluster

### PR DESCRIPTION
**Pull request by skovbakke**
Deploy kafka mirrormaker to sync the specified list of topics to the
target cluster. Nedded for integration with long term data storage
solution.

**Description of your changes:**

Checklist:

* [x] I have bumped the chart version
* [x] I have documented my changes if this is needed
* [x] I have described my changes in the "description of your changes" field
* [x] I have tested this change on a running cluster and the Argocd dashboard shows healthy and synced
* [ ] I have created the necessary tests in the chart, if they are possible/necessary
* [x] I have squashed commits if necessary

### CODE REVIEW

File: charts/kafka/chart/templates/kafka-mirrormaker.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{{ if .Values.dataexport.enable -}}
+---
+apiVersion: kafka.strimzi.io/v1beta2
+kind: KafkaMirrorMaker
```
Comment (klausenbusk) [inline charts/kafka/chart/templates/kafka-mirrormaker.yaml:4]: Maybe we should use Mirror Maker 2.0 🤔

https://strimzi.io/docs/operators/latest/overview.html#bidirectional_replication_across_two_clusters

**Review by MadsHT — DISMISSED** [review]
LGTM

File: charts/kafka/chart/templates/kafka-mirrormaker.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{{ if .Values.dataexport.enable -}}
+---
+apiVersion: kafka.strimzi.io/v1beta2
```
Comment (klausenbusk) [inline charts/kafka/chart/templates/kafka-mirrormaker.yaml:3]:
```suggestion
# https://strimzi.io/docs/operators/latest/overview.html#example_yaml_showing_mirrormaker_configuration
apiVersion: kafka.strimzi.io/v1beta2
```
or for Mirror Maker 2.0:
```suggestion
# https://strimzi.io/docs/operators/latest/overview.html#example_yaml_showing_mirrormaker_2_0_configuration
apiVersion: kafka.strimzi.io/v1beta2
```

**Comment by skovbakke** [comment]
I have changed the mirror maker instance to use MirrorMaker2.

**Review by jacobcrawford — CHANGES_REQUESTED** [review]
maybe we should set the replication factor to 2 instead of 1 wherever possible  so that we ensure  HA

**Comment by MadsHT** [comment]
Have we tested that MM2 is working as intended?

**Pull request closed without merging**